### PR TITLE
Added more logging to API view create

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Updated logging in create review step

### CODE REVIEW

File: eng/common/scripts/Create-APIReview.ps1
Comment (mitchdenny) [inline eng/common/scripts/Create-APIReview.ps1:125]: I think it would be better to have an ```aka.ms``` link here to documentation so you've got the chance to intercept folks with frequently asked questions (you know its going to happen 😄).

Comment (praveenkuttappan) [reply eng/common/scripts/Create-APIReview.ps1:125]: Thanks. I am working on updating FAQ as well as creating a new DL for arch board approvers based on feedback from archboard to be included in logs if developer needs emergency approval from arch board team.

Comment (mitchdenny) [reply eng/common/scripts/Create-APIReview.ps1:125]: No worries. I'll approve this now in anticipation of an aka.ms link. I didn't have any other feedback.

**Review by mitchdenny — APPROVED** [review]

File: eng/common/scripts/Create-APIReview.ps1
Comment (weshaggard) [inline eng/common/scripts/Create-APIReview.ps1:137]:
```suggestion
    # Detailed log for each scenario is mentioned above
```

Should we just have those location exit 1 instead of falling through?

File: eng/common/scripts/Create-APIReview.ps1
Comment (weshaggard) [inline eng/common/scripts/Create-APIReview.ps1:119]: Instead of directly putting an email address here it is probably just better to refer to the faq.

**Review by weshaggard — APPROVED** [review]

File: eng/common/scripts/Create-APIReview.ps1
Comment (weshaggard) [inline eng/common/scripts/Create-APIReview.ps1:49]: Should we exit 1 here?

Comment (weshaggard) [reply eng/common/scripts/Create-APIReview.ps1:49]: I see this is a reusable function. However given you are calling Write-Error in some contexts like in DevOps that will cause the script to stop.

Comment (praveenkuttappan) [reply eng/common/scripts/Create-APIReview.ps1:49]: I have changed it to Write-Host and let it fall through calling code.

**Pull request merged**
